src/plugins/intel_gpu/include/intel_gpu/primitives/crop.hpp

```diff
@@ -14,6 +14,15 @@ namespace cldnn {
 /// @addtogroup cpp_primitives Primitives
 /// @{
 
+/// @brief Select original ngraph op mode for the @ref crop layer.
+enum class crop_op_mode : int32_t {
+    none,
```

Comment:
Not critical comment, but, can we rename it as "standalone" or "default" because none sounds like nothing .. hmm

Reply:
Okay, I will change to crop_ngraph_op_mode.

Reply:
Done.